backend/routes/group.js

```diff
@@ -26,21 +26,32 @@ const Group = require("../models/group");
 const utils = require("../utils");
 const systemLogger = require("../logger.js").systemLogger;
 
-router.get("/", middlewares.issue.canView, listGroups);
-router.get("/:uid", middlewares.issue.canView, findGroup);
-router.put("/:uid", middlewares.issue.canCreate, updateGroup);
-router.post("/", middlewares.issue.canCreate, createGroup);
-router.delete("/:id", middlewares.issue.canCreate, deleteGroup);
+router.get('/groups/', middlewares.issue.canView, listGroups);
+router.get('/groups/:uid', middlewares.issue.canView, findGroup);
```

Comment:
We should get rid of these. And if it's head then make it call /revision/master/head so it's inline with every other end point

Reply:
So both Line 29 and 30 should have `revision/master/head/` inserted?

Reply:
yea, anything with out revision tagged should be revision/master/head i think